cmd/remove.go

```diff
@@ -0,0 +1,97 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/bitswan-space/bitswan-gitops-cli/internal/caddyapi"
+	"github.com/spf13/cobra"
+)
+
+func newRemoveCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "remove <gitops-name>",
+		Short:        "bitswan-gitops remove",
+		Args:         cobra.ExactArgs(1),
+		SilenceUsage: true,
+		Run: func(cmd *cobra.Command, args []string) {
+			gitopsName := args[0]
+			fmt.Printf("Removing Gitops: %s...\n", gitopsName)
+			err := removeGitops(gitopsName)
+			if err != nil {
+				fmt.Printf("Error removing gitops: %v\n", err)
+				return
+			}
+			fmt.Printf("Gitops %s removed successfully!\n", gitopsName)
+		},
+	}
+	return cmd
+}
+
+func removeGitops(gitopsName string) error {
+	bitswanPath := os.Getenv("HOME") + "/.config/bitswan/"
+
+	// 1. Ask for confirmation
```

Comment:
Before we ask for confirmation we should check if any automations are running in the workspace and list them out. We should then give the user the option to leave those automations running or get rid of them. When we get rid of them, we should also get rid of the caddy entries.